src/device-registry/bin/www

```diff
@@ -24,7 +24,7 @@ const ObjectId = mongoose.Types.ObjectId;
 const eventSchema = Joi.object({
     "s2_pm2_5": Joi.number().optional(),
     "s2_pm10": Joi.number().optional(),
-    "longitude": Joi.number().precision(5).greater(0).optional(),
+    "longitude": Joi.number().precision(5).optional(),
```

Comment:
update input validations so that values with no decimal places can also be allowed